src/components/NFTBook/ItemCard.vue

```diff
@@ -153,6 +153,15 @@
                 iscnWorkAuthor
               }}</span>
             </li>
+            <li
+              v-if="iscnWorkPublisher"
+              class="flex flex-col justify-center min-w-0"
+            >
+              <NFTBookSpecTableLabel :text="$t('identity_type_publisher')" />
+              <span :class="['font-[600]', displayNameStyle]">{{
+                iscnWorkPublisher
+              }}</span>
```

Comment:
Can be replaced with `<NFTBookSpecTableValue />`

Reply:
also updated iscn author above